package.json

```diff
@@ -58,13 +58,13 @@
     "@commitlint/cli": "17.0.3",
     "@commitlint/config-conventional": "17.0.3",
     "@cspell/cspell-types": "6.1.2",
-    "@storybook/addon-actions": "6.4.21",
-    "@storybook/addon-docs": "6.4.21",
-    "@storybook/addon-essentials": "6.4.21",
-    "@storybook/addon-links": "6.4.21",
-    "@storybook/node-logger": "6.4.21",
-    "@storybook/preset-create-react-app": "4.1.0",
```

Comment:
We aren't using this create react app preset are we?

Reply:
Nope. I'll kill it!

Reply:
We're not using the logger or actions either.